integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r4/EncounterFhirResourceProviderIntegrationTest.java

```diff
@@ -814,6 +822,62 @@ public void shouldThrow404WhenUpdatingNonExistingVisitAsJson() throws Exception
 		assertThat(response.getContentAsString(), notNullValue());
 	}
 	
+	@Test
+	public void shouldPatchExistingMedicationUsingJsonMergePatch() throws Exception {
```

Comment:
@mherman22 I missed this when reviewing. Could you submit a new PR renaming these methods?

Reply:
changes are part of https://github.com/openmrs/openmrs-module-fhir2/pull/506